src/ebay-page-notice/README.md

```diff
@@ -43,6 +44,8 @@ import "@ebay/skin/page-notice";
 
 | Name         | Type                                                                          | Stateful | Description                                                                                                                                 | Default       |
 | ------------ | ----------------------------------------------------------------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
-| `status`     | String ( `"general"`,`"attention"`, `"confirmation"`, or `"information"`) | No       | Determines the style and type of notice to be displayed                                                                                     | `"general"` |
+| `status`     | String ( `"general"`,`"attention"`, `"confirmation"`, or `"information"`) | No       | Determines the style and type of notice to be displayed                                                                                         | `"general"`   |
```

Comment:
this change is unnecessary as well

Reply:
This is just a whitespace change. Makes things line up more nicely in the editor.